skbio/util/_decorator.py

```diff
@@ -7,6 +7,9 @@
 # ----------------------------------------------------------------------------
 
 from __future__ import absolute_import, division, print_function
+from warnings import warn
```

Comment:
Can you import these packages instead of importing pieces from them? In general we're trying to avoid these types of imports because it breaks the package namespace and makes it less clear where a function came from.

```python
import textwrap
import warnings

import decorator # this is 3rd party so put in its own section
```